### PR TITLE
[FR] Week 14 + README + A few corrections

### PR DESCRIPTION
**Pull request by lbourdois**
My penultimate PR if everything goes well :)

### CODE REVIEW

File: docs/fr/README-FR.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+-->
+
+Ce répertoire a maintenant un [site web associé](https://atcold.github.io/pytorch-Deep-Learning/), où tout le matériel de cours peut être trouvé au format vidéo et textuel.
+
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic -->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md)
+[FR](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/fr/README-FR.md)
```
Comment (Atcold) [inline docs/fr/README-FR.md:16]: Use 🇫🇷    please and `&nbsp;` like for the other flags.

File: docs/fr/week14/14.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+lang: fr
+lang-ref: ch.14
+title: Week 14
```
Comment (Atcold) [inline docs/fr/week14/14.md:4]: Translate Week in French.

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to update *all* the `README-XX.md`. Also, use the flag and the same spacing character.

File: docs/fr/week14/14.md (hunk 2 above)
Comment (lbourdois) [reply docs/fr/week14/14.md]: Done

File: docs/fr/README-FR.md (hunk 1 above)
Comment (lbourdois) [reply docs/fr/README-FR.md]: Done. I also add arabic path for some file where it was missing

**Review by Atcold — APPROVED** [review]
Great, thanks!

**Pull request merged**